public/_redirects

```diff
@@ -70,3 +70,8 @@ https://www.asyncapi.io/* https://www.asyncapi.com/:splat 301!
 # Additional redirection
 /community/meetings /community/events 301!
 /cheatsheet https://github.com/asyncapi/website/tree/master/cheatsheet 302!
+
+# SHORT-DOC-URL-REDIRECTION:START
+/s/example https://github.com/asyncapi/spec/blob/master/examples/streetlights-mqtt-asyncapi.yml 301!
```

Comment:
It should be `302` (temporary redirect), as we might want to change the destination of short URLs. `301` would ask the browser to cache destination URL.